publisher-sdk/src/main/java/com/criteo/publisher/DependencyProvider.java

```diff
@@ -744,6 +750,35 @@ public MraidMessageHandler provideMraidMessageHandler() {
     return new MraidMessageHandler();
   }
 
+  @NonNull
+  public MraidController provideMraidController(
+      MraidPlacementType placementType,
+      AdWebView adWebView
+  ) {
+    if (placementType == MraidPlacementType.INLINE) {
+      return new CriteoBannerMraidController(
+          (CriteoBannerAdWebView) adWebView,
+          provideRunOnUiThreadExecutor(),
+          provideVisibilityTracker(),
+          provideMraidInteractor(adWebView),
+          provideMraidMessageHandler()
+      );
+    } else {
+      return new CriteoInterstitialMraidController(
+          provideRunOnUiThreadExecutor(),
```

Comment:
[NIT] Have the parameters in both constructors for `CriteoBannerMraidController` and `CriteoInterstitialMraidController` in the same order.

For ex `AdWebView,RunOnUiThreadExecutor,VisibilityTracker,MraidInteractor,MraidMessageHandler`.